tfjs-backend-cpu/BUILD.bazel

```diff
@@ -29,6 +30,17 @@ nodejs_test(
     tags = ["ci"],
 )
 
+tfjs_web_test(
+    name = "tfjs-backend-cpu_browser_test",
+    srcs = [
+        "//tfjs-backend-cpu/src:tfjs-backend-cpu_test_bundle",
+    ],
+    args = [],
+    browsers = [],
+    headless = False,
+    presubmit_browsers = []
```

Comment:
Nit: If this is empty, these tests won't run in presubmits. Is that intentional? Do the tests only work with `headless = False`?

Reply:
Done. Thanks! Also removes core tests from this test target.